docs/content/users/quickstart.md

```diff
@@ -176,18 +176,18 @@ Environment variables will be automatically added to your `.env` file to simplif
     ddev config --project-type=craftcms --docroot=web
 
     # Boot the project and install the starter project:
-    ddev start
     ddev composer create-project --no-scripts craftcms/craft
-    ddev craft install/craft \
+
+    # Install Craft CMS:
+    ddev craft install \
         --username=admin \
         --password=Password123 \
         --email=admin@example.com \
-        --site-name='$DDEV_PROJECT' \
```

Comment:
This is awkard:
- `--site-name` should be the "friendly name" of a project
- `$DDEV_PROJECT` (not the resolved value) will get persisted to `config/project.yaml`, which you don't want.

Best to just let the prompt ask them, or hardcode it